File: Merge two strings/Merge two strings.cpp
string merge (string S1, string S2)
{
    int n=min(S1.length(),S2.length());
    string s="";
    for(int i=0;i<n;i++){
        s=s+S1[i]+S2[i];
    }
    if(S1.length()>S2.length()){
        for(int i=n;i<S1.length();i++){
            s+=S1[i];
        }
    }else{
        for(int i=n;i<S2.length();i++){
            s+=S2[i];
        }
    }
    return s;
}

File: Merge two strings/Merge two strings.py
class Solution:
    def merge(self, S1, S2):
        res=""
        i=0
        while i<len(S1) or i<len(S2):
            if i<len(S1):
                res += S1[i]
            
            if i<len(S2):
                res += S2[i]
            
            i=i+1
            
        return res
